clients/vscode/src/Commands.ts

```diff
@@ -223,7 +225,19 @@ export class Commands {
       commands.executeCommand("editor.action.inlineSuggest.trigger");
     },
     "inlineCompletion.accept": () => {
+      const editor = window.activeTextEditor;
+      if (!editor) {
+        return;
+      }
+
       commands.executeCommand("editor.action.inlineSuggest.commit");
+
+      const uri = editor.document.uri;
+      const range = this.inlineCompletionProvider.getCurrentDisplayRange();
+      if (!range) {
+        return;
+      }
```

Comment:
```suggestion

      const range = this.inlineCompletionProvider.getCurrentDisplayRange();
      await commands.executeCommand("editor.action.inlineSuggest.commit");

      const uri = editor.document.uri;
      if (!range) {
        return;
      }
```

Reply:
Should it `await` the edit action of accept inline completion?

Reply:
The await causes us to be unable to get the current display item because it is no longer available. I moved the part that retrieves the edited range before committing the inline suggestion. The range should always exist if a code prediction is present.